app/src/main/java/comcmput301f16t01/github/carrier/Users/LoginActivity.java

```diff
@@ -45,17 +51,26 @@ private void tryQuickLogin() {
 
         UserController uc = new UserController();
 
-        if ( !uc.logInUser( username ) ) {
-            EditText usernameEditText = (EditText) findViewById( R.id.EditText_username );
-            usernameEditText.setText( username );
-            AlertDialog.Builder adb = new AlertDialog.Builder( this );
-            String message = "Your account '" + username + "' was not found!";
-            adb.setTitle( "Warning!" );
-            adb.setMessage( message );
-            adb.setPositiveButton( "OK", null );
+        if (ConnectionChecker.isThereInternet()) {
```

Comment:
This is about line 37. could your java doc comments be more descriptive? more than one line if possible?